userguide/storage.rst

```diff
@@ -1198,8 +1198,8 @@ available when editing multiple disks.
    |                              | menu      |            | `AAM <https://en.wikipedia.org/wiki/Automatic_acoustic_management>`__.                                                   |
    |                              |           |            |                                                                                                                          |
    +------------------------------+-----------+------------+--------------------------------------------------------------------------------------------------------------------------+
-   | Enable S.M.A.R.T.            | checkbox  | ✓          | Enabled by default if the disk supports S.M.A.R.T. Unsetting this option will disable any configured                     |
-   |                              |           |            | :ref:`S.M.A.R.T. Tests` for the disk.                                                                                    |
+   | Enable S.M.A.R.T.            | checkbox  | ✓          | Enabled by default when the disk supports S.M.A.R.T. Unsetting disables any configured disk :ref:`S.M.A.R.T. Tests` and  |
```

Comment:
Using the word "disk" here makes this confusing--is it (configured disk) tests or configured (disk tests)?
The word "configured" is part of the problem.  Let's talk on this and see if we can make it clearer.